.github/stale.yml

```diff
@@ -13,9 +13,20 @@ exemptLabels:
 staleLabel: stale
 # Comment to post when marking as stale. Set to `false` to disable
 markComment: >
+  Thanks for your contribution!
+
   This issue has been automatically marked as stale because it has not had
-  recent activity. It will be closed if no further activity occurs. Thank you
-  for your contributions.
+  recent activity. Because the Atom team treats their issues
+  [as their backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog), stale issues
+  are closed. If you would like this issue to remain open:
+
```

Comment:
❤️ I feel like that was a big issue for people.  Even though it was implied that more activity would keep it open, I like us being explicit and saying "here's what you can do if you want this to stay open".